library/qs_source/unstable/qsharp.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Should the unstable library be directly inside the `library` folder instead of `qs_source`?

Reply:
Fixed!